util/cron/test-perf.hpe-cray-ex-ofi.1node.bash

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env bash
+#
+# Run performance tests on an HPE Cray EX
```

Comment:
One last nit, maybe update this comment to indicate its for single node (`-nl 1`) runs